--- a/vendor/magento/framework/Image/Adapter/Gd2.php	2022-06-02 09:27:41.495452883 +0700
+++ b/vendor/magento/framework/Image/Adapter/Gd2.php	2022-06-02 09:28:55.430344121 +0700
@@ -343,7 +343,7 @@
         // fill image with indexed non-alpha transparency
         $transparentColor = false;

-        if ($transparentIndex >= 0 && $transparentIndex <= imagecolorstotal($this->_imageHandler)) {
+        if ($transparentIndex >= 0 && $transparentIndex < imagecolorstotal($this->_imageHandler)) {
             list($red, $green, $blue) = array_values(imagecolorsforindex($this->_imageHandler, $transparentIndex));
             $transparentColor = imagecolorallocate($imageResourceTo, (int) $red, (int) $green, (int) $blue);
         }
